### ISSUE TITLE
Issue in Hands on lab (Before the HOL): Task 4, step 3 (git config user.email/name)

### ISSUE THREAD

**Issue by EstebanZ** [microsoft/MCW-Cloud-native-applications#309]
Hi, when the subscription is new, in Cloud shell Git does not have name and mail configured, so when I run `bash create_azure_resources.sh` I get the following error:

![image](https://user-images.githubusercontent.com/9389266/155804838-dd13e6a5-0eeb-4931-a2c6-2552fb79a5b4.png)

Workaround: run before, (maybe this steps should be added in 'Before the HOL')

```
git config --global user.email "user@domain"
git config --global user.name "User Name"
```

**Comment by DawnmarieDesJardins** [comment]
@EstebanZ - What Exercise and task are you working on?

**Comment by hatboyzero** [comment]
This should be resolved in #311
